lib/storage/bdev.py

```diff
@@ -215,7 +215,8 @@ def Create(cls, unique_id, children, size, spindles, params, excl_stor,
     # create an optimally-striped volume; in that case, we want to try
     # with N, N-1, ..., 2, and finally 1 (non-stripped) number of
     # stripes
-    cmd = ["lvcreate", "-L%dm" % size, "-n%s" % lv_name]
+    # Use -Wn to not wipe signatures as trying to do so may fail on debian11
```

Comment:
I think the comment here can be a bit more descripting, something along the lines:

```
# When run non-interactively, newer LVM versions will fail (unless `--yes` is specified) when an existing filesystem signature is encountered while creating a new LV. Using `-Wn` disables this check.
```

The same should probably be mentioned in the commit message as well. Note that this is not a debian-specific issue, it's a change in the behavior of `lvcreate` in LVM2 >= 2.03.10.

Reply:
Thanks for your suggestion. I added the comment